prime/src/components/borrow/actions/UniswapAddLiquidityActionCard.tsx

```diff
@@ -302,7 +306,7 @@ export default function UniswapAddLiquidityActionCard(props: ActionCardProps) {
 
         if (!isToken0Selected) price = 1.0 / price;
         const nearestTick = roundDownToNearestN(
-          priceToTick(price, token0.decimals, token1.decimals),
+          priceToClosestTick(price, token0.decimals, token1.decimals),
           tickInfo.tickSpacing
         );
```

Comment:
I'm glad you were able to factor out your logic into `priceToClosestTick`, but the reasoning for this still isn't clear to me. We're already `floor`ing or `ceil`ing the resultant tick value with `roundDownToNearestN` and `roundUpToNearestN`, so why is the `priceToClosestTick` thing necessary in the first place?

Like aside from whether this is the right fix, what bug are you addressing in the first place?

Reply:
priceToTick was having issues with rounding, causing it to oftentimes return the tick above/below the actual closest tick. This caused issues with changing and updating ticks not working properly, as it sometimes skips over valid ticks (calculated by incrementing/decrementing the ticks) due to rounding. As for the outermost rounding, I am not sure this is doing much, but you can enlighten me on that as I believe you added it back in the day.

Reply:
#436 covers it